src/udev/udev-builtin-net_setup_link.c

```diff
@@ -19,7 +19,10 @@ static int builtin_net_setup_link(sd_device *dev, int argc, char **argv, bool te
                 return log_device_error_errno(dev, SYNTHETIC_ERRNO(EINVAL), "This program takes no arguments.");
 
         r = link_get_driver(ctx, dev, &driver);
-        if (r >= 0)
+        if (r < 0)
+                log_device_full_errno(dev, r == -EOPNOTSUPP ? LOG_DEBUG : LOG_WARNING,
```

Comment:
maybe use ERRNO_IS_NOT_SUPPORTED()? after all this might be an error the drivers generate, and they might each pick a slightly different error?

Reply:
In my testing, it's only `lo` that doesn't provide the value (and it returns EOPNOTSUPP). So... I think it's better to only silence the error for EOPNOTSUPP, since we know it's being returned by at least one driver. ERRNO_IS_NOT_SUPPORTED casts a fairly wide net, also stuff like EAFNOSUPPORT/EPFNOSUPPORT/ESOCKTNOSUPPORT which I wouldn't expect here. I don't want to inadvertently silence an unexpected error.